src/targets/gpu/include/migraphx/gpu/miopen.hpp

```diff
@@ -75,21 +75,31 @@ inline tensor_descriptor make_tensor(const migraphx::shape& os, bool pack = fals
     return t;
 }
 
+inline void insert_attribute_dim(std::vector<int>& padding,
```

Comment:
Maybe we can rename it to insert_attribute_1d_dims

Reply:
fixed